observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -8,6 +8,112 @@ toc::[]
 
 A connector connects two pipelines. It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
 
+[id="count-connector_{context}"]
+== Count Connector
+
+The Count Connector counts trace spans, trace span events, metrics, metric data points, and log records in exporter pipelines and using the following default metric names:
+
+* `trace.span.count`
```

Comment:
The metric names are self explanatory. I leave it to the docs team to see if we should add it as a table like the upstream [docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/countconnector#default-configuration  ).